Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js

```diff
@@ -0,0 +1,59 @@
+// call the displayWindowProperties on page loads
+document.addEventListener("DOMContentLoaded", function () {
+  displayWindowProperties();
+});
+
+function displayWindowProperties() {
+  // get the userAgent property from the window properties
+  const userAgent = window.navigator.userAgent;
+  console.log("displayWindowProperties ~ userAgent:", userAgent);
+
+  // get the screenWidth property from the window properties
+  const screenWidth = window.screen.width;
+  console.log("displayWindowProperties ~ screenWidth:", screenWidth);
+
+  // get the screenHeight property from the window properties
+  const screenHeight = window.screen.height;
+  console.log("displayWindowProperties ~ screenHeight:", screenHeight);
+
+  // get the currentUrl property from the window properties
+  const currentUrl = window.location.href;
```

Comment:
an single const can be created for currentURL and other location properties

Reply:
const currentUrlAndPath = `${window.location.href} - ${window.location.pathname}`;